### PR TITLE
Fixed #120:'uniminmax' scaling error for columns containing one single numeric value only. Added corresponding tests that confirm the error is not thrown anymore.

### LINKED ISSUE #120: ggparcoord: "uniminmax" scaling error when data frame columns contain only one single value
Dear ggally developers,

I found the above descriped issue when I attempted to plot a parallel coordinate plot using a data frame that contained columns that stored only one single value for all observations / rows.

Since I programmed a quick no bells and whistles fix I'll submit a pull request with the solution that worked for me. Maybe it is useful or maybe you can cherrypick something out of it.

Thank you for the work you put into developing and maintaining this package. It helps me a lot.

Best regards,
Christoph

### PR DESCRIPTION
**Pull request by schmidtchristoph**
Fixes issue #120.

### CODE REVIEW

File: R/ggparcoord.R [hunk 1]
```diff
@@ -309,9 +309,18 @@ ggparcoord <- function(
   inner_rescaler <- function(x, type = "sd", ...) {
     # copied directly from reshape because of import difficulties :-(
     # rescaler.data.frame
-    continuous <- sapply(x, is.numeric)
+    continuous    <- sapply(x, is.numeric)
     if (any(continuous)) {
-      x[continuous] <- lapply(x[continuous], inner_rescaler_default, type = type, ...)
+      if (type %in% c("sd", "robust", "range")) {
+        singleVal    <- sapply(x, function(col){if(length(unique(col)) == 1) TRUE else FALSE}) # indicating columns containing only one single value
+        isFactor     <- as.vector(sapply(x, is.factor))
```
Comment (schloerke) [inline R/ggparcoord.R]: Why the check for is.factor?  Wouldn't is.numeric cover this case?

**Comment by schloerke** [comment]
Nice change!

File: R/ggparcoord.R (hunk 1 above)
Comment (schmidtchristoph) [inline R/ggparcoord.R]: You are right. I removed this in a new commit. I guess it was left over when I played with columns storing a single factor. I didn't meant to litter the code.

**Comment by schmidtchristoph** [comment]
I just made a new commit that removed the unnecessary is.factor() check. Thanks Barret for pointing this out.

File: R/ggparcoord.R [hunk 2]
```diff
@@ -2,9 +2,7 @@ if(getRversion() >= "2.15.1") {
   utils::globalVariables(c("variable", "value", "ggally_splineFactor"))
 }
 
-##### KNOWN BUGS #####
-# - It does not currently work to pass groupColumn one of the variables that is being plotted
-#   as an axis
```
Comment (schloerke) [inline R/ggparcoord.R]: Ha.  it does work now.  Thank you

**Comment by schloerke** [comment]
moving to #122

**Comment by schloerke** [comment]
No problem!  Sometimes there's an odd check that needs to be made. Didn't know if I was missing something.

**Comment by schloerke** [comment]
merged

**Pull request closed without merging**